### ISSUE TITLE
Feature: Add an option to block the creation of a wish list when a user registers

### ISSUE THREAD

**Issue by Thulkash** [cmintey/wishlist#175]
**Is your feature request related to a problem? Please describe.**
Every time a user registers, a wish list is created for them. It would be nice of we can block this behavior for particular groups.

**Describe the solution you'd like**
Add an option to block this behavior

**Comment by cmintey** [comment]
Curious what your use case is for having a user without a list?

**Comment by cmintey** [comment]
Makes sense. I will be adding support for creating multiple lists and also the renaming support. I'll consider this feature as well, but with the multiple lists, you'll be able to delete them, including the default one
